docs/pages/extensibility/CustomTasks.md

```diff
@@ -13,24 +13,25 @@ You can configure your build process with additional build task. These custom ta
 To hook your custom tasks into the different build phases of a project, they need to reference other tasks to be executed before or after. This can be a [standard task](../Builder.md#standard-tasks) or another custom task. Note that a custom task will only be executed if the referenced task is executed (i.e. is not disabled).
 
 In the below example, when building the library `my.library` the custom `babel` task will be executed before the standard task `generateComponentPreload`.  
-Another custom task called `generateMarkdownFiles` is then executed immediately after the standard task `uglify`.
+Another custom task called `renderMarkdownFiles` is then executed immediately after the standard task `uglify`.
```

Comment:
uglify was removed in v3, should we use another task here?

Reply:
or lets keep it. We'll have another PR providing all the v3 branch documentation updates

Reply:
Done

Reply:
Done